modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation in the revision template, or by modifying the value for `containerConcurrency` in the Knative Serving ConfigMap.
```

Comment:
Do we need to document the ConfigMap stuff somewhere? Something for later?

Reply:
:joy: you told me not to yet when we initially discussed what to include, so I didn't. I am confused about containerConcurrency though - is this ConfigMap or not? Earlier in this doc you mentioned it's revision template?

Reply:
I'm only referring to a ConfigMap because this sentence does (and others do too I believe). You're right that this sentence is actually wrong, the containerConcurrency value is not set in the ConfigMap.

Reply:
We agreed to do this later during initial discussions unless there's a reason we'd add it now? The upstream docs on this need a overhaul tbh once I have time because the ConfigMap vs revision template thing is confusing from knative.dev - I didn't know it as well when I initially wrote that.